notifier.rst

```diff
@@ -0,0 +1,561 @@
+                    // Sent notifications to Slack and use Telegram if
+                    // Slack errored
+                    'main' => '%env(SLACK_DSN)% || %env(TELEGRAM_DSN)%',
+
+                    // Always sent notifications to both Slack and Telegram
+                    'all' => '%env(SLACK_DSN)% && %env(TELEGRAM_DSN)%',
+                ],
+            ],
+        ]);
+
+Creating & Sending Notifications
+--------------------------------
+
+To send a notification, autowire the
+:class:`Symfony\\Component\\Notifier\\NotifierInterface` (service ID
+``notifier``). This class has a ``send()`` method that allows you to send a
+:class:`Symfony\\Component\\Notifier\\Notification\\Notification` to a
+:class:`Symfony\\Component\\Notifier\\Recipient\\Recipient`::
+
+    // src/Controller/InvoiceController.php
+    namespace App\Controller;
+
+    use Symfony\Component\Notifier\NotifierInterface;
+    use Symfony\Component\Notifier\Notification\Notification;
+    use Symfony\Component\Notifier\Recipient\AdminRecipient;
+
+    class InvoiceController extends AbstractController
+    {
+        /**
+         * @Route("/invoice/create")
+         */
+        public function create(NotifierInterface $notifier)
+        {
+            // ...
+
+            // create a Notification that has to be send
+            // using the "email" channel
+            $notification = (new Notification('New Invoice', ['email']))
+                ->content('You got a new invoice for 15 EUR.');
+
+            // The receiver of the Notification
+            $recipient = new AdminRecipient(
+                $user->getEmail(),
+                $user->getPhonenumber()
+            );
+
+            // Send the notification to the recipient
+            $notifier->send($notification, $recipient);
+
+            // ...
+        }
+    }
+
+The ``Notification`` is created by using two arguments: the subject and
+channels. The channels specify which channel (or transport) should be used
+to sent the notification. For instance, ``['email', 'sms']`` will sent
+both an email and sms notification to the user. It is required to specify
+the transport when using chatters (e.g. ``['email', 'chat/telegram']``).
+
+The default notification also has a ``content()`` and ``emoji()`` method to
+set the notification content and icon.
+
+Symfony provides three types of recipients:
+
+:class:`Symfony\\Component\\Notifier\\Recipient\\NoRecipient``
+    This is the default and is usefull when there is no need to have
+    information about the receiver. For example, the browser channel uses
+    the current requests's :ref:`session flashbag <flash-messages>`;
+
+:class:`Symfony\\Component\\Notifier\\Recipient\\Recipient``
+    This contains only the e-mailadres of the user and can be used for
+    messages on the email and browser channel;
+
+:class:`Symfony\\Component\\Notifier\\Recipient\\AdminRecipient``.
+    This can contain both e-mailadres and phonenumber of the user. This
```

Comment:
Before you wrote “emailaddress” now “e-mail...”

Anyway a missing “s” but should be consistent

Reply:
I've decided to use `email address`, to be consistent with the mailer docs. Updated it everywhere in this article.